Forge.TreeWalker/contracts/ForgeSchemaDictionaryValidationRules.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "object",
```

Comment:
>type": "object [](http://example.com/codeflow?start=5&length=14)

(Moving this comment for better visibility)

We actually have a backlog item to remove "Label" as a required property in ChildSelector. Let's go ahead and do that in this PR since we're already messing with the validations rules a bit. This is on line 178 of ForgeSchemaValidationRules.json. Thanks!